tasks/compare-compiled-output.js

```diff
@@ -136,7 +136,7 @@ async function processComponent(
 	if (!component) return Promise.reject("No component specified.");
 
 	cleanAndMkdir(join(output, "diffs", component));
-	cleanAndMkdir(join(output, "latest"));
+	cleanAndMkdir(join(pathing.latest, component));
```

Comment:
This minor fix corrects parallelization errors that were occurring in the compiled output tool